server/root.go

```diff
@@ -66,11 +219,11 @@ func ServeCommands(cdc *codec.Codec, registerRoutesFn func(*lcd.RestServer)) *co
 }
 
 // RegisterRoutes register routes of all modules
-func RegisterRoutes(rs *lcd.RestServer) {
```

Comment:
Why are we changing this?

Reply:
That RestServer struct is imported from Cosmos SDK. For us, to be able to shut down the REST server properly, we needed access to the listener field, which is private in this case.
Not copying the RestServer struct from the package gave us control, and as an added bonus we got that logger that was also private.